lib/ui/hooks.dart

```diff
@@ -4,6 +4,20 @@
 
 part of dart.ui;
 
+@pragma('vm:entry-point')
+void _addView(
+  int id,
+) {
```

Comment:
Does the arguments need to be on multiple lines? It looks like this fits under 80 characters:

```suggestion
void _addView(int id) {
```